Sources/Sentry/SentryOptions.m

```diff
@@ -44,8 +44,12 @@ @implementation SentryOptions {
 {
     // The order of integrations here is important.
     // SentryCrashIntegration needs to be initialized before SentryAutoSessionTrackingIntegration.
+    // And SentrySessionReplayIntegration before SentryCrashIntegration.
```

Comment:
Yeah. Not good. We need to rethink integration install and running process.

Reply:
I already took one crack at refactoring the whole process, but it wound up getting huge. I'm still thinking of how it can be done one integration at a time.